macosx/README.md

```diff
@@ -0,0 +1,23 @@
+# Building a Mac OS X Package
+
+
+To build the distribution package for Gregorio for Mac, you first need to install [Packages](http://s.sudre.free.fr/Software/Packages/about.html) and [Pandoc](johnmacfarlane.net/pandoc/index.html).  Once those are installed, simply run:
+
+`create_pacakge.sh`
+
+This script contains all the necessary commands to create the distribution package.  It will first build and "install" the package into the `macosx` directory (this way we know where everything is).  It will then use Pandoc to convert the necessary documentation files from markdown to ??? (the format required by Packages).  Once that's done, it will use the command line interface for Packages to build the installer package.
```

Comment:
I think `macosx` should be `build/` and `??? (the format required by Packages)` should be `html`